modules/accessing-inference-endpoints-for-models-deployed-on-single-model-server.adoc

```diff
@@ -95,5 +95,9 @@ curl -ks <inference_endpoint_url>/v2/models/<model_name>/infer -d '{ "model_name
 --
 endif::[]
 
-//[role='_additional-resources']
-//.Additional resources
+[role='_additional-resources']
+== Additional resources
+* link:https://github.com/IBM/text-generation-inference[Text Generation Inference Server]
```

Comment:
Let's add the TGIS acronym here: 

* link:https://github.com/IBM/text-generation-inference[Text Generation Inference Server (TGIS)^]